trie/triedb/pathdb/nodebufferlist.go

```diff
@@ -602,10 +600,13 @@ func (w *proposedBlockReader) Node(owner common.Hash, path []byte, hash common.H
 		current = current.next
 	}
 
+	w.nf.baseMux.RLock()
 	node, err := w.nf.base.node(owner, path, hash)
```

Comment:
put `w.nf.baseMux.RUnlock()` after line 604 to avoid write it twice

Reply:
Fixed.